getting-started/kubernetes/vpp/getting-started.md

```diff
@@ -0,0 +1,199 @@
+Trying out the tech preview will give you a taste of these benefits and an opportunity to give feedback to the VPP dataplane team.
+
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **calico/node**
+- **VPP dataplane**
+
+### Concepts
+
+#### VPP
+
+The Vector Packet Processor (VPP) is a high-performance, open-source userspace network dataplane written in C, developed under the [fd.io](https://fd.io) umbrella. It supports many standard networking features (L2, L3 routing, NAT, encapsulations), and is easily extensible using plugins. The VPP dataplane uses plugins to efficiently implement Kubernetss services load balancing and {{site.prodname}} policies.
+
+### How to
+
+This guide details two ways to install Calico with the VPP dataplane:
+- On a [managed EKS cluster](#install-calico-with-the-vpp-dataplane-on-an-eks-cluster). This is the option that requires the least configuration.
+- On [any Kubernetes cluster](#install-calico-with-the-vpp-dataplane-on-any-kubernetes-cluster).
+
+In both cases, here are the details of what you will get:
+
+{% include geek-details.html details='Policy:Calico,IPAM:Calico,CNI:Calico,Overlay:IPIP,Routing:BGP,Datastore:Kubernetes' %}
+
+### Install Calico with the VPP dataplane on an EKS cluster
+
+#### Requirements
+
+For these instructions, we will use `eksctl` to provision the cluster. However, you can use any of the methods in {% include open-new-window.html text='Getting Started with Amazon EKS' url='https://docs.aws.amazon.com/eks/latest/userguide/getting-started.html' %}
+
+Before you get started, make sure you have downloaded and configured the {% include open-new-window.html text='necessary prerequisites' url='https://docs.aws.amazon.com/eks/latest/userguide/getting-started-eksctl.html#eksctl-prereqs' %}
+
+#### Provision and configure the cluster
+
+1. First, create an Amazon EKS cluster without any nodes.
+
+   ```bash
+   eksctl create cluster --name my-calico-cluster --without-nodegroup
+   ```
+
+2. Since this cluster will use {{site.prodname}} for networking, you must delete the `aws-node` daemon set to disable AWS VPC networking for pods.
+
+   ```bash
+   kubectl delete daemonset -n kube-system aws-node
+   ```
+
+3. Now that you have a cluster configured, you can install {{site.prodname}}.
+
+   ```bash
+   kubectl apply -f https://raw.githubusercontent.com/projectcalico/vpp-dataplane/master/yaml/generated/calico-vpp-eks.yaml
+   ```
+
+3. Finally, add nodes to the cluster.
+
+   ```bash
+   eksctl create nodegroup --cluster my-calico-cluster --node-type t3.medium --node-ami auto --max-pods-per-node 100
+   ```
+
+   > **Tip**: Without the `--max-pods-per-node` option above, EKS will limit the {% include open-new-window.html text='number of pods based on node-type' url='https://github.com/awslabs/amazon-eks-ami/blob/master/files/eni-max-pods.txt' %}. See `eksctl create nodegroup --help` for the full set of node group options.
+   {: .alert .alert-success}
+
```

Comment:
User will find DPDK after it is too late. So lets move it to step 3. 

3. Now that you have a cluster configured, you can install {{site.prodname}}.
   - Standard install
 ```bash
     kubectl apply -f https://raw.githubusercontent.com/projectcalico/vpp- 
  dataplane/master/yaml/generated/calico-vpp-eks.yaml
```
   - Install using DPDK driver for better networking performance
      ```bash
kubectl apply -f https://raw.githubusercontent.com/projectcalico/vpp-dataplane/master/yaml/generated/calico-vpp-eks-dpdk.yaml
```